### ISSUE TITLE
Hypohomocysteinemia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7715]
**Preferred term label:**
Hypohomocysteinemia

**Synonyms**

**Definition (free text, please give PubMed ID)**
A decreased concentration of homocystine in the blood.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
617744 
PMID: 29018201  3/4

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
